### ISSUE TITLE
Extraneous output of address when creating address beyond gap limit (2.6.4)

### ISSUE THREAD

**Issue by paxdickinson** [spesmilo/electrum#1828]
If I use addrequest with --force to create an address beyond the gap limit, electrum outputs the address before the JSON result. Note the extra output of the address before the JSON object. 

```
[admin@dev]$ electrum addrequest 0.0866817 --expiration 180 --force
1LbrNj8RBwsqrsSTfqUWrTc4D8b7Yqi4ey
{
    "URI": "bitcoin:1LbrNj8RBwsqrsSTfqUWrTc4D8b7Yqi4ey?amount=0.0866817", 
    "address": "1LbrNj8RBwsqrsSTfqUWrTc4D8b7Yqi4ey", 
    "amount": 8668170, 
    "amount (BTC)": "0.0866817", 
    "exp": 180, 
    "id": "7b617cc1ff", 
    "memo": "", 
    "status": "Pending", 
    "time": 1465567145
}
```

This is happening in account.py on line 70, I'm not a Python developer but it seems to me that function should not be outputting anything and that line should be removed.

**Comment by dabura667** [comment]
This is because it is generating a new address.

Electrum displays newly generated addresses to stdout.

This is unrelated to the addrequest command. It is because you generated a new address that electrum did not have cached in the wallet file.

**Comment by ecdsa** [comment]
fixed in f5cfa28710ccb661b61d80f40d94ad8ba6d25b9b
